README.md

```diff
@@ -1,368 +1,257 @@
-  statements in your code.
-- You can conditionally avoid running code that references document by passing in a boolean prop to your top level react
-  component. Since the passed in props Hash from the view helper applies to client and server side code, the best way to
-  do this is to use a generator function.
+This will make the data available in your component:
 
-You might do something like this in some file for your top level component:
 ```javascript
-global.App = () => <MyComponent serverSide={true} />;
+  this.props.initialName
 ```
 
-The point is that you have separate files for top level client or server side, and you pass some extra option indicating that rendering is happening server sie.
+### Understanding the Organization of the Generated Client Code
+The generated client code follows our organization scheme. Each unique set of functionality, is given its own folder inside of `client/app/bundles`. This encourages for modularity of DOMAINS. 
 
-## Optional Configuration   
+Inside of the generated "HelloWorld" domain you will find the following folders:
 
-Create a file `config/react_on_rails.rb` to override any defaults. If you don't specify this file,
-the default options are below.
++  `startup`: two types of files, one that return a container component and implement any code that differs between client and server code (if using server-rendering), and a `clientGlobals` file that exposes the aforementioned files (as well as a `serverGlobals` file if using server rendering). These globals files are what webpack is using as an entry point.
++ `containers`: "smart components" (components that have functionality and logic that is passed to child "dumb components").
++ `components`: includes "dumb components", or components that simply render their properties and call functions given to them as properties by a parent component. Ultimately, at least one of these dumb components will have a parent container component.
 
-The `server_bundle_js_file` must correspond to the bundle you want to use for server rendering.
+You may also notice the `app/lib` folder. This is for any code that is common between bundles and therefore needs to be shared (for example, middleware). 
 
-```ruby
-# Shown below are the defaults for configuration
-ReactOnRails.configure do |config|
-  # Client bundles are configured in application.js
-  # Server bundle is a single file for all server rendering of components.
-  config.server_bundle_js_file = "app/assets/javascripts/generated/server.js" # This is the default
+#### Additional Redux Folders
+If you have used the `--redux` generator option, you will notice the familiar additional redux folders in addition to the aforementioned folders. In this organization paradigm, each bundle has its own store. We do not set a global store and then use partial stores based off of that. Again, this is for bundle code modularity and isolation. 
```

Comment:
Note that if you want to reuse redux reducers across domains, then you will want to put the shared reducers under /client/app/lib